extension/options/options.html

```diff
@@ -81,6 +81,7 @@
         <select name="editor" id="editor">
           <option value="nvim">Neovim</option>
           <option value="vim">Vim</option>
+          <option value="emacsclient">Emacsclient</option>
```

Comment:
```suggestion
          <option value="emacsclient">EmacsClient</option>
```

https://www.emacswiki.org/emacs/EmacsClient uses an uppercase C.

Reply:
[Emacs itself uses "Emacs (Client)"](https://github.com/emacsmirror/emacs/blob/master/etc/emacsclient.desktop#L2), so let's go with that.